src/ios/RNSentry.m

```diff
@@ -1,299 +0,0 @@
-                    [RNSentryEventEmitter emitModuleTableUpdate:_self.moduleMapping.mutableCopy];
-                    [[NSUserDefaults standardUserDefaults] setObject:[NSString stringWithFormat:@"%lu", callNativeModuleAddress] forKey:@"RNSentry.__sentry_address"];
-                    [[NSUserDefaults standardUserDefaults] setObject:[RCTConvert NSString:param[@"__sentry_stack"]] forKey:@"RNSentry.__sentry_stack"];
-                    [[NSUserDefaults standardUserDefaults] synchronize];
-                } else {
-                    if (param != nil) {
-                        [newParams addObject:param];
-                    }
-                }
-            }
-        }
-        return SentrySWCallOriginal(bridge, module, newParams);
-    }), SentrySwizzleModeOncePerClassAndSuperclasses, key);
-}
-
-- (void)swizzleCallNativeModule:(Class)class {
-    static const void *key = &key;
-    SEL selctor = @selector(callNativeModule:method:params:);
-    uintptr_t callNativeModuleAddress = [class instanceMethodForSelector:selctor];
-
-    SentrySwizzleInstanceMethod(class,
-                                selctor,
-                                SentrySWReturnType(id),
-                                SentrySWArguments(NSUInteger moduleID, NSUInteger methodID, NSArray *params),
-                                SentrySWReplacement({
-        // TODO: refactor this block, its used twice
-        NSMutableArray *newParams = [NSMutableArray array];
-        if (params != nil && params.count > 0) {
-            for (id param in params) {
-                if ([param isKindOfClass:NSDictionary.class] && param[@"__sentry_stack"]) {
-                    [[NSUserDefaults standardUserDefaults] setObject:[NSNumber numberWithUnsignedInteger:callNativeModuleAddress] forKey:@"RNSentry.__sentry_address"];
-                    [[NSUserDefaults standardUserDefaults] setObject:[RCTConvert NSString:param[@"__sentry_stack"]] forKey:@"RNSentry.__sentry_stack"];
-                    [[NSUserDefaults standardUserDefaults] synchronize];
-                } else {
-                    if (param != nil) {
-                        [newParams addObject:param];
-                    }
-                }
-            }
-        }
-        return SentrySWCallOriginal(moduleID, methodID, newParams);
-    }), SentrySwizzleModeOncePerClassAndSuperclasses, key);
-}
-
-RCT_EXPORT_METHOD(clearContext)
-{
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0ul), ^{
-        [SentryClient.sharedClient clearContext];
-    });
-}
-
-RCT_EXPORT_METHOD(setLogLevel:(int)level)
-{
-    [SentryClient setLogLevel:[SentryJavaScriptBridgeHelper sentryLogLevelFromJavaScriptLevel:level]];
-}
-
-RCT_EXPORT_METHOD(setTags:(NSDictionary *_Nonnull)tags)
-{
-    SentryClient.sharedClient.tags = [SentryJavaScriptBridgeHelper sanitizeDictionary:tags];
```

Comment:
Sorry I realize this is months old now @HazAT but it appears this was removed in the new API and therefor impacts react-native ability to report context on native crashes, see #694 - I can push up a PR if this was accidental.

However, a bit confused on how that would work in this new API, seeing as `setTags` is just exported directly from `@sentry/core` - is it possibly to create some sort of proxy so native context is added as well?

Reply:
We will fix this, instead of setting all properties individually we will just persist the Scope.
Keep an eye on the issue.

Reply:
Thanks @HazAT